files/galaxy/dynamic_rules/usegalaxy/tool_destinations.yaml

```diff
@@ -701,6 +701,15 @@ mass_spectrometry_imaging_qc: {mem: 110}
 mass_spectrometry_imaging_filtering: {mem: 20}
 metaspades: {cores: 10, mem: 250}
 megahit: {cores: 10, mem: 80}
+megan_daa_meganizer:
+  env:
```

Comment:
I think you need to add `mem: 8` as well.

Reply:
Ah, ok. Thanks!

Reply:
or now 40